core/src/main/java/org/openstack4j/model/network/ext/builder/LbPoolV2Builder.java

```diff
@@ -84,4 +84,14 @@
      * @return LbPoolV2Builder
      */
     LbPoolV2Builder listenerId(String listenerId);
+
+    /**
+     * The ID of the load balancer under which this pool will be created.
+     * Each load balancer can have zero or more pools associated with it. These pools can be used for L7policies.
+     * Either listener_id or loadbalancer_id must be specified.
+     *
+     * @param loadbalancerId
+     * @return LbPoolV2Builder
+     */
+    LbPoolV2Builder loadbalancerId(String loadbalancerId);
```

Comment:
Unfortunately, one more instance of the lowercase `loadbalancerId`.

Reply:
fixed the field format and in its implementations, tests as well.